job-server/src/spark.jobserver/WebApi.scala

```diff
@@ -51,6 +53,17 @@ class WebApi(system: ActorSystem,
 
   val myRoutes = jarRoutes ~ contextRoutes ~ jobRoutes ~ healthzRoutes ~ otherRoutes
 
+  lazy val authenticator: AuthMagnet[AuthInfo] = {
+    if (config.hasPath("shiro.authentication") &&
+      config.getBoolean("shiro.authentication")) {
```

Comment:
sure, didn't know what your general policy was on this.
application.conf now says "shiro.authentication = off", removed hasPath in WebApi